quickstarts/angular1/01-login.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Setup and Authentication
+name: Angularjs-login
+alias:
+  - angular
+  - angularjs
+language:
+  - Javascript
+framework:
+  - AngularJS
+image: /media/platforms/angular.png
+tags:
+  - quickstart
+  - angular
+snippets:
+  dependencies: /quickstarts/angular1/snippets/dependencies
+  init: /quickstarts/angular1/snippets/init
+  head: /quickstarts/angular1/snippets/head
+alias:
+  - angular
+---
+
+## Setup and Authentication
+
+One of the most aching aspects of an Angular App is authentication. It becomes worse when you have routes to protect, page refresh to deal with, tokens to manage, multi-factor authentication, and so on. This quickstart is planned to help you tackle these challenges.
+
+This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your Angular JS apps with Auth0.
+
+
+### Create an Application
+
+Create an [Auth0 account](https://manage.auth0.com/login) and an authentication application from the [dashboard](https://manage.auth0.com/#/). Once you create an app, you'll be provided with credentials (Domain, Client ID, and Client Secret) which should be stored somewhere safe (do not commit this information to your git repo!).
```

Comment:
Keep in mind that many people who are running this tutorial will 1) already have an auth0 account and 2) will already have an app. One thing you can do is check if they are logged in. It might be good to have two variants of this paragraph. 1) for anonymous users and 2) for users who have accounts.